week-01/assessment/capsuleHotel/src/Hotel.java

```diff
@@ -0,0 +1,142 @@
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class Hotel {
+
+    public static void main(String[] args) {
+        System.out.println("Welcome to the Capsules Hotel!");
+
+        Scanner console = new Scanner(System.in);
+
+        System.out.print("What is the number of Capsules? ");
+        int capsulesCount = 0;
+        capsulesCount = Integer.parseInt(console.nextLine());
+        System.out.println(capsulesCount);
+        String[] guestList = new String[capsulesCount];
+
+        System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+        int operation = console.nextInt();
+        boolean exitConfirmation = false;
+        while (operation != 4 || !exitConfirmation) {
+            switch (operation) {
+                case 1:
+                    boolean hasSuccessfullyCheckedIn = checkIn(console, guestList);
+                    while (!hasSuccessfullyCheckedIn) {
+                        hasSuccessfullyCheckedIn = checkIn(console, guestList);
```

Comment:
This is an interesting way to handle this. I haven't seen this before. I would have probably put this inside of your checkin method. However it's not incorrect here.